components/SerialServer/src/serial.c

```diff
@@ -70,13 +72,11 @@ const char *all_output_colours[2][MAX_CLIENTS] = {
     {
         /* Processed streams */
         ANSI_COLOR(RED),
-        ANSI_COLOR(GREEN),
```

Comment:
From the commit message: "and the "Green" color is removed from the Serial Server, as it is very difficult to read."
The rendered color for an ANSI color is affected by whatever local terminal emulator is being used (https://en.wikipedia.org/wiki/ANSI_escape_code#Colors). My opinion is that this entry shouldn't be removed for this reason. What do others think?

Reply:
If the Max clients is dropped to 10, we'd have to remove one color. Personal preference is Green because `screen` and TeraTerm don't render it well, but I could always modify the PR/commit message based on community feedback.

Reply:
I don't understand why "A side effect of this change is that the SerialServer is now limited to 10 clients" is required?

Reply:
Ok I see.